components/cam/bld/namelist_files/use_cases/20TR_cam5_av1c-04p2-MMF-1mom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0"?>
```

Comment:
The 20TR use cases are also unused and untested I believe.

Reply:
Aren't they used for AMIP runs?

Reply:
I mean, I know they aren't used by us currently

Reply:
removed